scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -291,12 +326,30 @@ function AIDriveStrategyUnloadCombine:getDriveData(dt, vX, vY, vZ)
         moveForwards = self:moveToNextFillNode()
     elseif self.state == self.states.MOVING_AWAY_FROM_UNLOAD_TRAILER then
         self:moveAwayFromUnloadTrailer()
+
+    ---------------------------------------------
+    --- Unloading on the field
+    ---------------------------------------------
+    elseif self.state == self.states.DRIVE_TO_FIELD_UNLOAD_POSITION then
+        self:setMaxSpeed(self:getFieldSpeed())
+    elseif self.state == self.states.PREPARE_FOR_FIELD_UNLOAD then
+        self:prepareForFieldUnload()
+    elseif self.state == self.states.UNLOADING_ON_THE_FIELD then
+        self:setMaxSpeed(self.settings.reverseSpeed:getValue())
+        --- For some reason this is a problem ?
```

Comment:
Search for didNotMoveTimer in the code and comments, this shouldn't be necessary.

Reply:
Oh, I see you found it, still, it is a general CP problem, a generic solution would be better.

Reply:
@pvaiko Should we fix this in another PR later?

Reply:
Sure.